lib/has_content_id.rb

```diff
@@ -0,0 +1,14 @@
+module HasContentId
```

Comment:
I think I prefer `HasContentId` – in the GOV.UK stack, the `content_id` is a first class citizen that identifies the content everywhere (something like an ISBN) and in my eyes, has a place in the domain language.